explore/src/main/scala/explore/config/ManualSequenceTables.scala

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.config
+
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.core.model.sequence._
+import lucuma.ui.reusability._
+import react.common._
+import react.semanticui.elements.header.Header
+import react.semanticui.elements.segment.Segment
+
+final case class ManualSequenceTables(config: ManualConfig)
```

Comment:
This looks very similar to `GeneratedSequenceTable` could it be generated?

Reply:
They are very similar indeed. I expect them to diverge in the future. I'll unify them nonetheless in a next PR.